### PR TITLE
pr_template: Provides tips for cherry-pick

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
pr_template: Provides tips for cherry-pick

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -12,6 +12,13 @@
 
 <!-- Fill in "x" in [] to tick the checkbox below.-->
 
+**Tips for choosing the affected version(s):**
+
+- If your change supplement missing or incomplete content, or fixes inaccurate or wrong contents, **CHOOSE MASTER ONLY**.
+- If your change involves a version-specific feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.
```
Comment (qiancai) [inline .github/pull_request_template.md:18]: By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

**Review by qiancai — COMMENTED** [review]
@TomShawn PTAL

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -194,6 +194,21 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
...
+- 完善和优化文档内容，例如补充缺失或不完整的信息。
+- 修正不准确或错误的文档内容，例如默认值错误、描述不准确、示例错误、拼写错误等。
+- 重新组织现有文档的某个局部，例如“部署标准集群”、“数据迁移”、“TiDB 生态工具”等。
+
+如果你的 PR 改动符合以下任一情况，请**选择 dev 版以及受影响的 release 版本**：
+
+- 涉及与版本相关的功能行为变化。
+- 涉及与版本相关的兼容性变化，例如更改某个配置项或变量的默认值。
+- 修复文档页面的渲染或显示错误。
+- 修复文档内的死链
```
Comment (qiancai) [inline CONTRIBUTING.md:210]:
```suggestion
- 修复文档内的死链。
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (TomShawn) [reply .github/pull_request_template.md:18]:
```suggestion
By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.  
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -194,6 +194,21 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 | `/cc` | 将 PR 分配给指定的 reviewer，需 @指定用户的 GitHub ID，多个 GitHub ID 间用逗号分隔。 | `/cc @TomShawn, @yikeke` |
 | `/uncc` | 移除 PR 之前指定的 reviewer。  | `/uncc @TomShawn`|
 
+## 版本选择指南
+
+如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**。此 PR 的改动在合并后将显示到 https://docs.pingcap.com/zh/tidb/dev/，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```
Comment (qiancai) [inline CONTRIBUTING.md:199]:
```suggestion
如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**。此 PR 的改动在合并后将显示到 [官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -194,6 +194,21 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 | `/cc` | 将 PR 分配给指定的 reviewer，需 @指定用户的 GitHub ID，多个 GitHub ID 间用逗号分隔。 | `/cc @TomShawn, @yikeke` |
 | `/uncc` | 移除 PR 之前指定的 reviewer。  | `/uncc @TomShawn`|
 
+## 版本选择指南
+
+如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**。此 PR 的改动在合并后将显示到 [官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```
Comment (TomShawn) [inline CONTRIBUTING.md:199]:
```suggestion
如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**。此 PR 的改动在合并后将显示到[官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```

Comment (qiancai) [inline CONTRIBUTING.md]:
```suggestion
如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**。此 PR 的改动在合并后将显示到[官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -194,6 +194,21 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 | `/cc` | 将 PR 分配给指定的 reviewer，需 @指定用户的 GitHub ID，多个 GitHub ID 间用逗号分隔。 | `/cc @TomShawn, @yikeke` |
 | `/uncc` | 移除 PR 之前指定的 reviewer。  | `/uncc @TomShawn`|
 
+## 版本选择指南
+
+如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**。此 PR 的改动在合并后将显示到[官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```
Comment (TomShawn) [inline CONTRIBUTING.md:199]:
```suggestion
如果你的 PR 改动符合以下任一情况，推荐**只选择 dev 版**，即 PR 只合入 master 分支。此 PR 的改动在合并后将显示到[官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -194,6 +194,21 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
...
+
+- 完善和优化文档内容，例如补充缺失或不完整的信息。
+- 修正不准确或错误的文档内容，例如默认值错误、描述不准确、示例错误、拼写错误等。
+- 重新组织现有文档的某个局部，例如“部署标准集群”、“数据迁移”、“TiDB 生态工具”等。
+
+如果你的 PR 改动符合以下任一情况，请**选择 dev 版以及受影响的 release 版本**：
```
Comment (TomShawn) [inline CONTRIBUTING.md:205]:
```suggestion
如果你的 PR 改动符合以下任一情况，请**选择 dev 版以及受影响的 release 版本**，即同时改动 master 分支以及受影响的 release 分支：
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -91,7 +91,7 @@ TiDB 文档的修改需要遵循一定的流程，具体如下。考虑到有些
 
 > **注意：**
 >
-> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版）、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。
+> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版）、v5.1、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[版本选择指南](#版本选择指南)。
```
Comment (TomShawn) [inline CONTRIBUTING.md:94]:
```suggestion
> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版，对应 master 分支）、v5.1、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[版本选择指南](#版本选择指南)。
```

File: CONTRIBUTING.md (hunk 5 above)
Comment (qiancai) [inline CONTRIBUTING.md:199]:
```suggestion
如果你的 PR 改动符合以下任一情况，推荐**只选择 master 分支**。此 PR 的改动在合并后将显示到[官网文档 Dev 页面](https://docs.pingcap.com/zh/tidb/dev/)，在下一次 TiDB 发新版本时将显示到对应版本的文档页面。
```

File: CONTRIBUTING.md (hunk 6 above)
Comment (qiancai) [inline CONTRIBUTING.md:205]:
```suggestion
如果你的 PR 改动符合以下任一情况，请**选择 master 分支以及受影响的 release 分支**：
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -91,7 +91,7 @@ TiDB 文档的修改需要遵循一定的流程，具体如下。考虑到有些
 
 > **注意：**
 >
-> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版）、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。
+> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版，对应 master 分支）、v5.1、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[版本选择指南](#版本选择指南)。
```
Comment (qiancai) [inline CONTRIBUTING.md:94]:
```suggestion
> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版，对应文档仓库的 master 分支）、v5.1、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[版本选择指南](#版本选择指南)。
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -194,6 +194,21 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 | `/cc` | 将 PR 分配给指定的 reviewer，需 @指定用户的 GitHub ID，多个 GitHub ID 间用逗号分隔。 | `/cc @TomShawn, @yikeke` |
 | `/uncc` | 移除 PR 之前指定的 reviewer。  | `/uncc @TomShawn`|
 
+## 版本选择指南
+
```
Comment (qiancai) [inline CONTRIBUTING.md:198]:
```suggestion

创建 Pull Request 时，你需要在 Pull Request 的描述模版中选择文档改动适用的版本分支。

```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**